packages/go/analysis/analysis.go

```diff
@@ -104,16 +102,24 @@ func ClearSystemTags(ctx context.Context, db graph.Database) error {
 	})
 }
 
-func ParseKind(rawKind string) (graph.Kind, error) {
-	for _, adKind := range append(ad.Nodes(), ad.Relationships()...) {
-		if adKind.String() == rawKind {
-			return adKind, nil
-		}
-	}
+func ValidKinds() []graph.Kind {
+	var (
+		lenCalc = len(ad.Nodes()) + len(ad.Relationships()) + len(azure.NodeKinds()) + len(azure.Relationships())
+		kinds   = make([]graph.Kind, 0, lenCalc)
+	)
+
+	kinds = append(kinds, ad.Nodes()...)
+	kinds = append(kinds, ad.Relationships()...)
+	kinds = append(kinds, azure.NodeKinds()...)
+	kinds = append(kinds, azure.Relationships()...)
+
+	return kinds
+}
```

Comment:
Moving our list of valid kinds out of `ParseKind` helps us reuse this important bit without having to do weird stuff with the `ParseKind` function elsewhere.